Source/BoogieDriver/BoogieDriver-NetCore.csproj

```diff
@@ -13,7 +13,6 @@
     <AssemblyName>BoogieDriver</AssemblyName>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <PackageOutputPath>nupkg</PackageOutputPath>
```

Comment:
Why are we deleting this?

Reply:
I wanted to see where the package is placed by default and found it cleaner to have no overlap between the build output of different configurations, i.e., everything concerning a particular configuration is placed in `bin/${CONFIGURATION}/`. I'll add it back if you prefer.

Reply:
We’re considering builds with multiple configurations?

Reply:
Yes. Currently Travis is configured to build and test the Debug and Release configurations. The deployment should only happen for Release.

Reply:
But those are two separate builds, one configuration per build.

Wouldn’t it be nice to find the package in the same place independently of configuration?

Reply:
I was thinking of a local clone of the repository. The same argument could be made for the other build output. Putting everything into `bin` instead of `bin/Debug/netcore3.1` would make it easier to find the binaries (e.g., in test scripts). Still, I'm against that because it might be useful to have multiple configurations compiled simultaneously.

That being said, for the package I don't care.

Reply:
Right. For `bin` I agree, because, like you say, you might want to build both locally.

But these two lines are just about the package and related configuration for Travis, which only builds one configuration, and can be simplified accordingly.

Reply:
Then again, maybe deleting the line above is better than simplifying the line below.

Let’s keep your version :-).